state_transfer/transport/transport.go

```diff
@@ -39,8 +39,8 @@ type Transport interface {
 	// ServerVolumes returns a list of volumes transfers can add to their server Pods
 	ServerVolumes() []v1.Volume
 	Direct() bool
-	CreateServer(client.Client, endpoint.Endpoint) error
-	CreateClient(client.Client, endpoint.Endpoint) error
+	CreateServer(client.Client, string, endpoint.Endpoint) error
```

Comment:
what is the need for changing interfaces for prefix?

Reply:
So I needed to create a new stunnel for block syncing since I have a separate server pod. This required a new stunnel/service/route. To identify the difference I added a prefix. This way it is easy to tell which stunnel is for filesystem sync and which one is for block sync.